lib/components-schema-v1_8_x.ts

```diff
@@ -665,6 +665,27 @@ export const componentsDefinitionSchema_v1_8_x: JSONSchema7 = {
                 type: 'string',
             },
         },
+
+        customStyles: {
+            type: 'array',
+            description:
+                'List of custom styles definitions which can be configured in the look and feel style of an article',
+            items: {
+                type: 'object',
+                properties: {
+                    label: { type: 'string', minLength: 1, maxLength: 20 },
```

Comment:
The label is now defined in the component set definition. Should it be possible to translate this property?

Reply:
I'm ok with it not being translatable.